web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html

```diff
@@ -45,7 +45,7 @@
 
   Object.defineProperty(this, testProp, {
     get: function() { _propAccessCount++; },
-    enumerable: true
+    enumerable: true,
```

Comment:
@jgraham why no trailing commas in all object literals? Or the inclusion of this one?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas